tests/api/runtimes/test_nuclio.py

```diff
@@ -402,6 +403,26 @@ def test_compile_function_config_with_special_character_labels(
         ) = server.api.crud.runtimes.nuclio.function._compile_function_config(function)
         assert config["metadata"]["labels"].get(key) == val
 
+    def test_compile_function_config_with_node_selector(
```

Comment:
this is what you are actually testing
```suggestion
    def test_compile_function_config_node_selector_enriched_from_project(
```